README.md

```diff
@@ -24,6 +28,11 @@ Required: **NO**
 
 The name you want the file to have in Google Drive. If this input is not provided, it will use the `filename`.
 
+## ``overwrite``
+Required: **NO**
+
+If you want to overwrite the filename with existing file, it will use the `filename`.
```

Comment:
Given that `filename` is the file that is stored locally, and `name` is the preferred name to be displayed on google drive, shouldn't this support overwriting based on `name`? And if `name` is not provided, then use `filename`.

Using the `overwrite` parameter now rends `name` useless. This doesn't seem to be the preferred behaviour.